spec/dummy/app/graphql/dummy_schema.rb

```diff
@@ -3,7 +3,10 @@ class DummySchema < GraphQL::Schema
     query:            Types::QueryType,
     mutation:         Types::MutationType,
     resource_loaders: [
-      GraphqlDevise::ResourceLoader.new('User', only: [:login, :confirm_account]),
+      GraphqlDevise::ResourceLoader.new(
+        'User',
```

Comment:
Might be better to test with a model that didn't also mount the queries on a separate schema. So it fails without your change